### PR TITLE
test(draggable): refactor for cypress tests

### PR DESCRIPTION
**Pull request by DlgSHi**
### Proposed behaviour

Refactor for `draggable` component to use new cypress-component-react framework for testing.

### Current behaviour

Currently we are using the regular cypress approach with BDD.

### Checklist

- [x] Commits follow our style guide
- [ ] Related issues linked in commit messages if required
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required
- [ ] Typescript `d.ts` file added or updated if required

#### QA
- [ ] Tested in CodeSandbox/storybook
- [x] Add new Cypress test coverage if required
- [ ] Carbon implementation matches Design System/designs
- [ ] UI Tests GitHub check reviewed if required

### Testing instructions

- Run npx cypress open-ct to check if there is newly added test.file
 - [x] Check if the `draggable.test.js` file passed
 - [x] Compare all tests were moved/refactored from .feature file -> .test.js file
 - [x] Check if all unnecessary files were deleted, including .json file

### CODE REVIEW

File: src/components/draggable/draggable.test.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      </DraggableItem>
+      <DraggableItem key="4" id={4}>
+        <Checkbox label="Draggable Label Four" />
+      </DraggableItem>
+    </DraggableContainer>
+  );
+};
+
+context("Test for Draggable component", () => {
+  describe("Check functionality for Draggable component", () => {
```
Comment (ZhuoyuJin) [inline src/components/draggable/draggable.test.js:40]: describe("check functionality for Draggable component", () => {

Comment (DlgSHi) [reply src/components/draggable/draggable.test.js]: fixed

**Review by edleeks87 — APPROVED** [review]

File: src/components/draggable/draggable.test.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    console.log(items);
+    if (getOrder) {
+      getOrder();
+    }
+  };
+
+  return (
+    <Box>
+      <DraggableContainer mb={9} getOrder={handleUpdate} {...props}>
+        <DraggableItem key="4" id={4}>
```
Comment (sianford) [inline src/components/draggable/draggable.test.js:51]: Should this first DraggableItem be `key="5"` `id={5}` and the others increment from that if it's the start of a block of different containers?

Comment (DlgSHi) [reply src/components/draggable/draggable.test.js:51]: it doesn't matter :) it is just an example for tests :)

**Review by ZhuoyuJin — APPROVED** [review]

**Review by sianford — DISMISSED** [review]

**Review by sianford — APPROVED** [review]

**Review by ZhuoyuJin — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Pull request merged**